benchmarks/text_generation.py

```diff
@@ -20,11 +20,6 @@
 from tensorflow import keras
```

Comment:
We need to update this README with new metrics: https://github.com/keras-team/keras-nlp/blob/master/benchmarks/README.md